src/redux/books/books.js

```diff
@@ -1,27 +1,47 @@
+import { getAllBooks, postBook, deleteBook } from '../../api/api';
+
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const GET_BOOKS = 'bookStore/books/GET_BOOKS';
 
 const initialState = [];
 
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
+const convertDataObjectToArray = (data) => {
+  const dataArray = [];
+
+  Object.keys(data).map((key) => {
```

Comment:
The project is unable to run due to the below error. Please make sure the app is running and has no error.
![type-error](https://user-images.githubusercontent.com/35504149/145223083-a5bcf11f-4863-4ea8-9055-98cd4fa137d8.png)